doc/ref_model/chapters/chapter07.md

```diff
@@ -172,6 +171,12 @@ The platform supports the workload, and in effect controls access to the workloa
 * Ensure that all the platform's components (including hypervisors, VMs, etc.) are kept up to date with the latest patch.
 * In order to tightly control access to resources and protect them from malicious access and introspection, Linux Security Modules such as SELinux should be used to enforce access rules.
 
+**Vulnerability Management**
+* Security defect must be reported.
+* Cadence should aligned with Cloud Infrastructure vendors (OSSA for OpenStack).
+* Components must be analysed: mechanisms to validate components of the platform stack by checking libraries and supporting code against the Common Vulnerabilities and Exposures (CVE) databases, https://cve.mitre.org/, to determine whether the code contains any known vulnerabilities must be embedded into the NFVI architecture itself.  Some of the components required include tools for checking common libraries against CVE databases integrated into the deployment and orchestration pipelines.
```

Comment:
```suggestion
* Components must be analysed: mechanisms, to validate components of the platform stack by checking the code libraries and all other code against the Common Vulnerabilities and Exposures (CVE) [databases](https://cve.mitre.org/) for the presence of any known vulnerabilities, must be embedded into the Cloud Infrastructure.  Some of the required mechanisms include tools for checking common libraries against CVE databases integrated into the deployment and orchestration pipelines.
```

Reply:
@pgoyal01 Thanks, I have added your suggestion into the last commit modifying the whole paragraph. Please review